docs/resources/samples/convert-csv.md

```diff
@@ -0,0 +1,86 @@
+     */
+    data.push(value.match(/(?:,|\n|^)("(?:(?:"")*[^"]*)*"|[^",\n]*|(?:\n|$))/g));
+  });
+
+  // Put the data in the worksheet.
+  let sheet = workbook.getWorksheet("Sheet1");
+  let range = sheet.getRangeByIndexes(0, 0, data.length, data[0].length);
+  range.setValues(data);
+
+  // Add any formatting or table creation that you want.
+}
+```
+
+## Power Automate flow: Create new .xlsx files
+
+1. Sign into [Power Automate](https://flow.microsoft.com) and create a new **Scheduled cloud flow**.
+1. Set the flow to **Repeat every** "1" "Day" and select **Create**.
+1. Get the template Excel file. This is the basis for all the converted .csv files. Add a **New step** that uses the **OneDrive for Business** connector and the **Get file content** action. Provide the file path to the "Template.xlsx" file.
+    * **File**: /output/Template.xlsx
+1. Rename the **Get file content** step by going the **...** menu of that step (in the upper right corner of the connector) and selecting the **Rename** option. Change the step name to "Get Excel template".
+
+     :::image type="content" source="../../images/convert-csv-flow-1.png" alt-text="The completed OneDrive for Business connector in Power Automate, renamed to be Get Excel template.":::
+1. Get all the files in the "output" folder. Add a **New step** that uses the **OneDrive for Business** connector and the **List files in folder** action. Provide the folder path that contains the .csv files.
+    * **Folder**: /output
+
+    :::image type="content" source="../../images/convert-csv-flow-2.png" alt-text="The completed OneDrive for Business connector in Power Automate.":::
+1. Add a condition so that the flow only operates on .csv files. Add a **New step** that is the **Condition** control. Use the following values for the **Condition**.
+    * **Choose a value**: *Name* (dynamic content from **List files in folder**). Note that this dynamic content has multiple results, so an **Apply to each** *value* control surrounds the **Condition**.
+    * **ends with** (from the dropdown list)
+    * **Choose a value**: .csv
+
+    :::image type="content" source="../../images/convert-csv-flow-3.png" alt-text="The completed Condition control with the Apply to each control around it.":::
+1. The rest of the flow is under the **If yes** section, since we only want to act on .csv files. Get an individual .csv file by adding a **New step** that uses the **OneDrive for Business** connector and the **Get file content** action. Use the **Id** from the dynamic content from **List files in folder**.
+    * **File**: *Id* (dynamic content from the **List files in folder** step)
+1. Rename the new **Get file content** step to "Get .csv file". This helps distinguish this file from the Excel template.
+1. Make the new .xlsx file, using the Excel template as the base content. Add a **New step** that uses the **OneDrive for Business** connector and the **Create file** action. Use the following values.
```

Comment:
As discussed offline, we are seeing different versions of Power Automate. My version only has "Add an action" as an option inside If yes. If yours says "New step", maybe we should make this direction more ambiguous? 
```suggestion
1. Make the new .xlsx file, using the Excel template as the base content. Choose **Add an action** and set it to use the **OneDrive for Business** connector and the **Create file** action. Use the following values.
```